src/Nethermind/Nethermind.Trie/TrieNodeResolverWithReadFlags.cs

```diff
@@ -23,8 +23,13 @@ public TrieNode FindCachedOrUnknown(Keccak hash)
         return _baseResolver.FindCachedOrUnknown(hash);
     }
 
-    public byte[]? LoadRlp(Keccak hash)
+    public byte[]? LoadRlp(Keccak hash, ReadFlags flags = ReadFlags.None)
```

Comment:
Cosmetic - change name from flags to overridenFlags? Wdyt?

Reply:
Can change `_flags` to `_defaultFlags` how about that?

Reply:
Much better. I've already changed during rereview :)